src/printapp/static/js/app.js

```diff
@@ -140,10 +140,35 @@ App.SignInController = Ember.Controller.extend({
 });
 
 App.QueueController = Ember.ArrayController.extend({
-  itemController: 'queueItem'
+  needs: ['application','queue'],
+  itemController: 'queueItem',
+
+  actions : {
+
+    // TODO: use "item.loading = true" and read that in the view to decide when to show the loading spinner.
+    deleteJob: function(job_id) {
+      var controller = this;
+      var queue = controller.get('model');
```

Comment:
You can use `this` instead of `controller` here. You only need to alias `this` so that it can be accessed from closures (nested functions, like the callback functions for ajax requests on line 157.